css/styles.css

```
@@ -318,26 +376,77 @@ p {
 
 .phase-items {    
     list-style: none;
-    font-family: Poppins, Helvetica, sans-serif;
     font-size: 18px;
-    font-weight: 400;
     line-height: 28px;
     letter-spacing: 0;
-    text-align: left;
     color: #c2c3c5;
+    text-transform: capitalizeupp;
```

Looks like there is a typo, right?

```suggestion
    text-transform: capitalize;
```

or you wanted to make it CAPS?

```suggestion
    text-transform: uppercase;
```